manager/integration/tests/common.py

```diff
@@ -6111,3 +6111,13 @@ def wait_for_instance_manager_count(client, number, retry_counts=120):
         time.sleep(RETRY_INTERVAL_LONG)
 
     return len(ims)
+
+
+def wait_delete_dm_device(api, name):
+    for i in range(RETRY_COUNTS):
+        path = os.path.join("/dev/mapper" + name)
```

Comment:
```suggestion
        path = os.path.join("/dev/mapper", name)
```

Reply:
And suggest move this outside of the loop.

Reply:
Updated. Thank you.